lib/components/src/tabs/tabs.tsx

```diff
@@ -204,10 +204,10 @@ Tabs.displayName = 'Tabs';
   bordered: false,
 };
 
-type FuncChilden = () => void;
+type FuncChildren = () => void;
```

Comment:
Technically `-en` is a plural marker, as is `-r`, but English is quirky, so we went with `children` not `childr` nor `childen`.